examples/messages.vroom

```diff
@@ -7,10 +7,16 @@ the command that causes them if you want it to work.
 This mechanism allows vroom to know what lines errors occur on. You can match
 multiple messages on one command.
 
+  :set cmdheight=99
+
   :echomsg "First" | echomsg "Second"
   ~ First
   ~ Second
 
+Note sometimes multiple lines of output cause vroom to get stuck at a "Hit ENTER
+to continue" prompt, and increasing cmdheight is a simple workaround to avoid
+getting stuck.
+
```

Comment:
Can we have cmdheigt=99 for neovim in the config?

I remember fixing this in some other place and I don't think a comment in vroom examples is going to save people time when debugging this.

Reply:
The context is in #83. I'd prefer not to bury workarounds even deeper and fix it for 99% of cases bc then it's that much more mysterious when it does fail. Really we should have a way for the neovim mod to detect if it's stuck at a "Press ENTER" prompt and do something intelligent, but that might require rethinking the whole neovim integration to use remote screen integration (#100).

OTOH, we might be able to give a much nicer error for neovim timeouts in general with pointers towards common troubleshooting suggestions. WDYT about that approach?

Reply:
SGTM, yes.
Burying workarounds is not a good idea, I agree.